python/tskit/_version.py

```diff
@@ -1,4 +1,4 @@
 # Definitive location for the version number.
 # During development, should be x.y.z.devN
 # For beta should be x.y.zbN
-tskit_version = "0.3.1"
+tskit_version = "0.3.1.dev1"
```

Comment:
I guess it should be 0.3.2.dev1 - otherwise the current development version will be consider older than the released one?

Reply:
Gah, of course. This line of code is my nemesis.